tools/build_langserver/lsp/lsp_test.go

```diff
@@ -423,26 +422,18 @@ func TestCompletionFunction(t *testing.T) {
 	err = h.Request("textDocument/completion", &lsp.CompletionParams{
 		TextDocumentPositionParams: lsp.TextDocumentPositionParams{
 			TextDocument: lsp.TextDocumentIdentifier{
-				URI: "file://test/test.build",
+				URI: testURI,
 			},
 			Position: lsp.Position{
-				Line:      1,
+				Line:      0,
 				Character: 6,
 			},
 		},
 	}, completions)
 	assert.NoError(t, err)
 	assert.Equal(t, &lsp.CompletionList{
 		IsIncomplete: false,
-		Items: []lsp.CompletionItem{
-			{
-				Label:            "go_library",
-				Kind:             lsp.CIKFunction,
-				InsertTextFormat: lsp.ITFPlainText,
-				TextEdit:         textEdit("rary", 1, 6),
-				Documentation:    h.builtins["go_library"].Stmt.FuncDef.Docstring,
-			},
-		},
```

Comment:
lps (build_langserver) doesn't yet support plugins like go-rules

Reply:
Oh rly? I guess we should sort that out then.

Reply:
Yep, really. Updated this test to use a builtin (plugin_repo).